File: android-sdk-ui/src/main/java/com/braze/ui/activities/ContentCardsActivity.java
package com.braze.ui.activities;

import android.os.Bundle;

import com.appboy.ui.R;
import com.braze.ui.contentcards.ContentCardsFragment;

/**
 * The {@link ContentCardsActivity} in an Activity class that displays the Braze Content Cards
 * Fragment. This class can be used to integrate Content Cards as an Activity.
 *
 * Note: To integrate Braze Content Cards as a Fragment instead of an Activity, use the
 * {@link ContentCardsFragment} class.
 */
public class ContentCardsActivity extends BrazeBaseFragmentActivity {
  @Override
  public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.com_braze_content_cards_activity);
  }
}

File: android-sdk-ui/src/main/java/com/braze/ui/activities/ContentCardsActivity.kt
package com.braze.ui.activities

import android.os.Bundle
import com.braze.ui.R

/**
 * The [ContentCardsActivity] in an Activity class that displays the Braze Content Cards
 * Fragment. This class can be used to integrate Content Cards as an Activity.
 *
 * Note: To integrate Braze Content Cards as a Fragment instead of an Activity, use the
 * [ContentCardsFragment] class.
 */
open class ContentCardsActivity : BrazeBaseFragmentActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.com_braze_content_cards_activity)
    }
}
